### DISCUSSION TITLE
Speaker Diariasation performance

### DISCUSSION THREAD

**Discussion by arrrrr3186** [pyannote/pyannote-audio#1701] [General]
Hello @hbredin @mogwai @joonaskalda,

I'm currently working with a speaker diarization model tailored for Indian accents, but it's not performing as expected in certain scenarios. I'm really close to getting the desired output. 

Could you advise on how to fine-tune or train the pre-existing model specifically for Indian accents? Alternatively, any other suggestions for improving the output would be greatly appreciated.

Regards,
Anuj

**Comment by arrrrr3186** [comment]
Hello @hbredin @mogwai @joonaskalda,

if anyone of you can guide me on the above the issue, would be great help to me.

Regards,
Anuj

**Comment by mogwai** [comment]
https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#how-can-i-improve-performance
